Makefile.am

```diff
@@ -282,6 +285,7 @@ BATS = \
 	test/functional/update/update-client-certificate.bats \
 	test/functional/update/update-download.bats \
 	test/functional/update/update-fail-to-get-mom.bats \
+	test/functional/update/update-ignore-also-add-flag.bats \
```

Comment:
I think the name of this test is not correct, "update-ignore-also-add-flag", also-add is not a flag.

Reply:
you're right. I'll rename that